### DISCUSSION TITLE
Send audio data to Bluetooth Earbuds

### DISCUSSION THREAD

**Discussion by RickLloyd24** [pschatzmann/arduino-audio-tools#1625] [Q&A] [answered]
Thank for the great library.  I am using an ESP32 WROOM and would like to send audio to some TOZO earbuds.  It seems the best example is "communication-ble-server-send.ino".

When I compile the example,, I get the following 2 error messages.

C:\Users\rickl\OneDrive\Documents\Arduino\libraries\arduino-audio-tools\src/Sandbox/BLE/AudioBLEServerESP32.h:111:16: error: 'class BLECharacteristic' has no member named 'notifyAudioChange'
     info_char->notifyAudioChange();
                ^~~~~~~~~~~~~~~~~
C:\Users\rickl\OneDrive\Documents\Arduino\libraries\arduino-audio-tools\src/Sandbox/BLE/AudioBLEClientESP32.h:259:18: **error:** 'class BLERemoteCharacteristic' has no member named 'registerFornotifyAudioChange'; did you mean 'registerForNotify'?
       info_char->registerFornotifyAudioChange(notifyCallback);

                  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
                  registerForNotify

Any help or suggestions would be greatly appreciated.

**Comment by pschatzmann** [comment, marked as answer]
I think you are confusing here something: The ESP32 does not support any official BLE Audio Standard

**Reply by RickLloyd24** [reply]
I thought I could do it in software.  Still learning Bluetooth.

**Reply by pschatzmann** [reply]
It would work if you have headphones that support Bluetooth A2DP. 

BLE Audio is not working...

**Comment by RickLloyd24** [comment]
The earbud manufacturer says they support A2DP.  Do you know why I am getting the compile errort?

**Comment by pschatzmann** [comment]
I committed a correction: you will need to install the updated version...

**Comment by RickLloyd24** [comment]
Thank you, I will try to get it to work.

**Reply by pschatzmann** [reply]
You will need to use the A2DP examples and not the BLE ones
